src/reactionsystem_conversions.jl

```diff
@@ -653,7 +653,7 @@ function Base.convert(::Type{<:JumpSystem}, rs::ReactionSystem; name = nameof(rs
     spatial_convert_err(rs::ReactionSystem, JumpSystem)
 
     (remove_conserved !== nothing) &&
-        error("Catalyst does not support removing conserved species when converting to JumpSystems.")
+        throw(ArgumentError("Catalyst does not support removing conserved species when converting to JumpSystems."))
```

Comment:
Ensuring that this is a `ArgumentError` allow us to write a better test later on (since we can test for that as opposed to `Exception`, which could be generated by other stuff).